### ISSUE TITLE
The Wither

### ISSUE THREAD

**Issue by Mercerenies** [Mercerenies/a-turtles-troll#152]
I don't think I had written this down yet, so I'm going to go ahead and summarize my thoughts here. I want the Wither to be an interesting post-Dragon boss, in the form of a boss rush.

* Per #142, wither spawn eggs will be disabled and will not spawn as a result of events.
* Once you defeat the ender dragon, I'll increase the drop rate of wither skulls significantly, so that you can summon the Wither more easily.
* The Wither himself summons other mobs as part of his attack pattern.
* At high HP, he summons weak common mobs like silverfish and bees.
* At medium-high HP, he adds in sturdier mobs like zombies.
* At medium-low HP, he adds in mobs that *themselves* spawn other mobs, like skeletons and blazes.
* When he gets low, he straight-up starts summoning boss mobs like the warden and ender guardians.
* And, of course, the Wither will passively heal during the whole fight.

**Comment by Flipsie27** [comment]
Will the Wither still be able to attack?

**Comment by Mercerenies** [comment]
@Flipsie27 Yes, it'll be like Blazes and Skeletons. They still get all their normal attacks but *also* get a new attack on top of it.

**Comment by Flipsie27** [comment]
I'm scared
